opentelemetry-zipkin/src/exporter/mod.rs

```diff
@@ -93,6 +93,9 @@ impl ZipkinExporterBuilder {
     }
 
     /// Assign client implementation
+    ///
+    /// Note: Programmatically setting the timeout will override any value
+    /// set via the environment variable `OTEL_EXPORTER_ZIPKIN_TIMEOUT`.
```

Comment:
this comment is for timeout, but this method is for setting client?

Reply:
correct, you can only set the timeout manually, if you create a client, because the timeout is used during the client creation.

Reply:
can you modify the comment to make it clearer?

_when using `with_http_client`, the timeout will depend on the provided client implementation, and may not respect timeout set via the environment variable `OTEL_EXPORTER_ZIPKIN_TIMEOUT`._